includes/admin.form.inc

```diff
@@ -9,6 +9,15 @@
  * Admin form.
  */
 function islandora_pathauto_admin_settings(array $form, array &$form_state) {
+  // Get possible query language types.
+  $backend_options = module_invoke_all('islandora_basic_collection_query_backends');
```

Comment:
This seems unnecessary and could cause confusion if there are additional collection backends that might not apply in this context. Would be more clear to just populate the select box with the 2 options Solr and SPAQL / RI, etc. Not to mention if the collection SP wasn't enabled, there'd be no options.